src/main/java/de/zalando/aruha/nakadi/config/JsonConfig.java

```diff
@@ -33,14 +39,15 @@
     @Bean
     @Primary
     public ObjectMapper jacksonObjectMapper() {
-        final ObjectMapper objectMapper = new ObjectMapper()
-                .setPropertyNamingStrategy(CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES);
+        final ObjectMapper objectMapper = new ObjectMapper();
 
+        objectMapper.setPropertyNamingStrategy(CAMEL_CASE_TO_LOWER_CASE_WITH_UNDERSCORES);
         objectMapper.registerModule(enumModule());
+        objectMapper.registerModule(eventTypeSchemaModule());
```

Comment:
String is allowed. More over object value is a convenience, string value is default